hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/TestLogAggregationService.java

```diff
@@ -324,7 +324,7 @@ public void testLocalFileRemainsAfterUploadOnCleanupDisable() throws Exception {
         this.remoteRootLogDir.getAbsolutePath());
     LogAggregationService logAggregationService = spy(
         new LogAggregationService(dispatcher, this.context, this.delSrvc, super.dirsHandler));
-    verifyLocalFileDeletion(logAggregationService);
+    verifyLocalFileDeletion(logAggregationService, 4321L);
```

Comment:
Is there some logic behind these numbers? If not I think we should generate random numbers instead, else someone adding a new tests, unaware of these numbers will landup into the same mess. But I am ok to do the present way if others feel this is better approach

Reply:
Thanks for the suggestion, I also think it would be better to use random numbers.